sql/table.h

```diff
@@ -3959,6 +3965,20 @@ inline bool is_user_table(TABLE *table) {
   return strncmp(name, tmp_file_prefix, tmp_file_prefix_length);
 }
 
+#ifdef WITH_WSREP
+inline bool is_wsrep_system_table(const char *db_name, size_t db_len,
+                                  const char *table_name, size_t table_len) {
+  return (WSREP_DB_NAME.length == db_len &&
+          !my_strcasecmp(system_charset_info, WSREP_DB_NAME.str,
+                         db_name) &&
+          WSREP_TABLE_PREFIX.length < table_len &&
+          !native_strncasecmp(table_name, WSREP_TABLE_PREFIX.str,
```

Comment:
why we use strcmp and strNcmp? 
I would prefer using strNcmp, but I see is_perfschema_db uses strcmp (so probably it is ensured by the caller that argument is the proper, null-terminated string).

At least I would stick to one of them if there is no reason for using different comparison functions.